addons/website_event_track/static/src/js/event_track_reminder.js

```diff
@@ -30,47 +36,107 @@ publicWidget.registry.websiteEventTrackReminder = publicWidget.Widget.extend({
     _onReminderToggleClick: function (ev) {
         ev.stopPropagation();
         ev.preventDefault();
-        var self = this;
-        var $trackLink = $(ev.currentTarget).find('i');
+        var $trackLink = $(ev.currentTarget);
```

Comment:
will only say it here (but to be applied on the file), we should take the opportunity to remove the jquery present here to use vanilla JS instead :slightly_smiling_face: